lua/gatekeeper/dylib.lua

```diff
@@ -171,3 +171,8 @@ function print_lls_dump_entry(lls_dump_entry, acc)
 		", ip: " .. ip .. ", mac: " .. ha ..
 		", port: " .. port_id .. "]\n"
 end
+
+function update_gt_lua_states_incrementally(gt_conf, lua_code)
+	dylib.l_update_gt_lua_states_incrementally(gt_conf,
+		string.dump(lua_code, True))
```

Comment:
1. Truth is Lua is `true`, lower cap T.

2. Add a comment to explain why you chose to strip the debug information. If the reason is to save space, it's better to drop the parameter because we need help when things go wrong. In addition, given that the code comes from the dynamic configuration block, the code is already limit to ~64KB due to the communication protocol.